packages/react-notion-x/src/block.tsx

```diff
@@ -568,7 +572,7 @@ export const Block: React.FC<BlockProps> = (props) => {
 
     case 'code': {
       if (block.properties.title) {
-        const content = block.properties.title[0][0]
```

Comment:
Could you change it so there isn't a function call? I'd prefer it in one line.

Reply:
Could be useful to have a `getBlockTitle` function in `notion-utils` that does this in a safe way. Not necessary, just a thought.

Reply:
ok, i will fix it.

Reply:
> 

you means create a function such as named `getCodeContent` to do this?

Reply:
`getBlockTitle` would probably take a block as input and do this `block.properties.title.map((arr) => { return arr[0] }).join('')`
probably need to use ?'s or have some checks that the title is there. But I'm guessing  thats what he means.
I'm fine either way.

Reply:
> getBlockTitle would probably take a block as input and do this block.properties.title.map((arr) => { return arr[0] }).join('')
probably need to use ?'s or have some checks that the title is there. But I'm guessing thats what he means.
I'm fine either way.

i got it. but i found that the function `getBlockTitle` will lost some **rich text formatting**, the code:
![image](https://user-images.githubusercontent.com/36353894/144699097-4eae339f-157e-4dbc-bab5-916193d8f008.png)

of course, generally speaking, there are no such `rich text characters` in the code, so the problem should not be big